### ISSUE TITLE
Graphical glitch on chatroom's user list

### ISSUE THREAD

**Issue by ghost** [smogon/pokemon-showdown#5016]
If you've a chatroom openned while battling, the chatroom's user list is glitched.
![bandicam 2018-12-08 22-18-44-363](https://user-images.githubusercontent.com/45188383/49693075-a7b17c80-fb37-11e8-90fe-3377dcb7e35c.jpg)

**Comment by Zarel** [comment]
...I'm sure the Preact rewrite will fix this.

**Comment by DanielProGames** [comment]
@Zarel What exactly is the Preact rewrite? I think a rewrite could fix this glitch. 
I also want to make 2 suggestions (since last 6 months, Pokémon Showdown has improved its features, I'm very glad for it):
1: Global FAQ commands.
2: Better the alt checking system. If Lemon and Apple joined PS with IP (148.112.233.202), then both, Lemon and Apple will be alts (Alts of Lemon: Apple), but if Lemon joined PS with IP (238.140.88.100), but Apple is still on IP (148.112.233.202), then Lemon's alts now are: Apple. But if Banana joined PS with IP (238.140.88.100), then Banana's alts now are: Lemon and Apple and viceversa.

**Comment by Zarel** [comment]
Huh, I got a PM asking for global FAQ commands. It might be a decent idea. @bumbadadabum?

I don't understand what you're asking about the alt checking system, though.

**Comment by DanielProGames** [comment]
Thank you, I'm the same who PMed you asking for that feature. And for the alt checking system, I'm making a proposal for improving its functionally (altrought I don't know its current functionally).

**Comment by Zarel** [comment]
Yes, I know you're requesting a change to the alt system. But I don't understand what you're asking to be changed. I also think you should probably understand how the current one works before asking for changes.

**Comment by DanielProGames** [comment]
Yes, you might be right; how the actual system works?

**Comment by Zarel** [comment]
...it's just a list of users on the same IP.

It also lists past usernames, for every account and user on the list.

**Comment by Slayer95** [comment]
@Zarel, it might not be too late to censor the IPs above.

**Comment by Zarel** [comment]
The only way to censor the IPs is to delete the post, and I don't think they're that sensitive. They certainly don't come from Main because the OP isn't staff on Main. They look like random numbers to me, actually.

**Comment by Slayer95** [comment]
Closing since this is fixed in https://play.pokemonshowdown.com/beta
